### PR TITLE
remove error wrapper type

### LINKED ISSUE #1863: Avoid `ErrorWrapper` type in main.rs
https://github.com/paritytech/polkadot/pull/1855#pullrequestreview-517981130

Currently there is a error wrapper type, that only exist to bridge the gap between `eyre` errors which are required to be `Send + Sync` and various error types and dyn and trait bounds which do not require it.

Long-term we shouldbjmp all those requirements to have the requirements tightened in all places to be `Send+Sync`

### PR DESCRIPTION
**Pull request by drahnr**
Companion https://github.com/paritytech/substrate/pull/7446 

Final cleanup to be able to remove the silly ErrorWrapper type, due to missing `Send+Sync` bounds.

~Not sure what to do with the `fn batch_all` weights impls that are now needed with the latest substrate.~

Fixes #1863

### CODE REVIEW

File: Cargo.toml
Comment (coriolinus) [inline Cargo.toml:117]:
```suggestion
"../scripts/packaging/polkadot.service" = { path = "/usr/lib/systemd/system/polkadot.service", mode = "644" }

```

Files should have trailing newlines.

File: runtime/kusama/src/weights/pallet_utility.rs
Comment (coriolinus) [inline runtime/kusama/src/weights/pallet_utility.rs:32]: I also have no idea what these `batch_all` impls are about; we should sort out what they mean and ensure that this is a reasonable impl before merging.

File: runtime/polkadot/src/weights/pallet_utility.rs
Comment (coriolinus) [inline runtime/polkadot/src/weights/pallet_utility.rs:32]:
```suggestion
	fn batch_all(c: u32) -> Weight {
```

**Review by coriolinus — APPROVED** [review]
Approve pending `batch_all` resolution of one sort or another.

**Review by ordian — APPROVED** [review]
Less code is always nice :)

**Comment by ordian** [comment]
```
error[E0277]: `(dyn sp_state_machine::error::Error + 'static)` cannot be shared between threads safely
  --> src/main.rs:25:12
   |
25 |     cli::run()?;
   |               ^ `(dyn sp_state_machine::error::Error + 'static)` cannot be shared between threads safely
```

**Comment by drahnr** [comment]
> ```
> error[E0277]: `(dyn sp_state_machine::error::Error + 'static)` cannot be shared between threads safely
>   --> src/main.rs:25:12
>    |
> 25 |     cli::run()?;
>    |               ^ `(dyn sp_state_machine::error::Error + 'static)` cannot be shared between threads safely
> ```

Just merged in polkadot master before the CI triggered. Not sure how to specify a dependency (or should this just become a companion too?).

**Comment by ordian** [comment]
> > ```
> > error[E0277]: `(dyn sp_state_machine::error::Error + 'static)` cannot be shared between threads safely
> >   --> src/main.rs:25:12
> >    |
> > 25 |     cli::run()?;
> >    |               ^ `(dyn sp_state_machine::error::Error + 'static)` cannot be shared between threads safely
> > ```
> 
> Just merged in polkadot master before the CI triggered. Not sure how to specify a dependency (or should this just become a companion too?).

even with b09c07b it doesn't compile unfortunately

**Comment by drahnr** [comment]
```
   Compiling polkadot v0.8.26 (/media/supersonic1t/projects/parity/polkadot)
    Finished dev [unoptimized + debuginfo] target(s) in 4m 51s
```
Not sure what's the delta that causes the different outcome..

**Pull request merged**
